webapp/lib/controller.dart

```diff
@@ -509,6 +531,10 @@ void initUI() {
       currentUserConfig = userConfig ?? currentUserConfig;
       var newConfig = currentUserConfig.applyDefaults(defaultUserConfig);
       applyConfiguration(newConfig);
+    }, (Object error) {
+      if (error is FirebaseError) {
```

Comment:
I don't think you need to check the type of error here and only catch `FirebaseError`s - unless I'm missing something?

Also, after we change the `onErrorListener` to take the stacktrace, I think you can just pass `showAndLogError` directly, without an anonymous callback

Reply:
Right. I've passed `showAndLogError` instead directly.